eclipse-scout-core/src/form/fields/numberfield/NumberFieldModel.ts

```diff
@@ -13,4 +13,14 @@ export interface NumberFieldModel extends BasicFieldModel<number, number | strin
   minValue?: number;
```

Comment:
Would be nice if you could add some jsdoc to the other properties as well to make it complete